core/src/scss/components/alert/_alert.scss

```diff
@@ -58,17 +58,32 @@
     font-size: 1.7rem;
 
     .su-far,
-    .su-fas {
+    .su-fas,
+    .fas {
       @include margin(null null null 0.5em);
     }
 
     i,
     .su-far,
-    .su-fas {
+    .su-fas,
+    .fas {
       font-style: normal;
       text-decoration: none;
     }
 
+    &.su-text-black {
+      &:hover {
+        background: $su-color-digital-blue-light;
+        color: $su-color-black;
+      }
+    }
+    &.su-text-white {
+      &:hover {
```

Comment:
Same here, please add support for :focus state

Reply:
Also, we have a file `._alert--text-light.scss` in the same folder as _alert.scss, so it would probably make sense to move this part to that file. And you could use the .su-alert--text-light class on the Giving site alerts with dark background/light text. Thanks, @andrewbaraniuk !